### DISCUSSION TITLE
8 bit quantization

### DISCUSSION THREAD

**Discussion by Positronx** [microsoft/onnxruntime-genai#514] [Ideas]
Hello,
Are 8 bit quantization methods planned to be supported by ONNXRuntime-genai in the future?
Thanks,

**Comment by mgiessing** [comment]
Yeah, int8 quantization would be of interest for me as well!

**Comment by prashantaithal** [comment]
Any update on this?

**Comment by elephantpanda** [comment]
I just tried it with the int4 phi-3 onnx model and it worked fine if that's any help.
